MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/labs/IndoorMapActivity.java

```diff
@@ -67,21 +68,24 @@ public void onMapReady(final MapboxMap mapboxMap) {
         levelButtons = findViewById(R.id.floor_level_buttons);
 
         boundingBox = new ArrayList<>();
-        boundingBox.add(Position.fromCoordinates(-77.03791, 38.89715));
-        boundingBox.add(Position.fromCoordinates(-77.03791, 38.89811));
-        boundingBox.add(Position.fromCoordinates(-77.03532, 38.89811));
-        boundingBox.add(Position.fromCoordinates(-77.03532, 38.89708));
 
-        mapboxMap.setOnCameraChangeListener(new MapboxMap.OnCameraChangeListener() {
-          @Override
-          public void onCameraChange(CameraPosition position) {
+        boundingBox.add(Point.fromLngLat(-77.03791, 38.89715));
+        boundingBox.add(Point.fromLngLat(-77.03791, 38.89811));
+        boundingBox.add(Point.fromLngLat(-77.03532, 38.89811));
+        boundingBox.add(Point.fromLngLat(-77.03532, 38.89708));
+
+        boundingBoxList = new ArrayList<>();
+        boundingBoxList.add(boundingBox);
 
-            if (position.zoom > 16) {
+        mapboxMap.addOnCameraMoveListener(new MapboxMap.OnCameraMoveListener() {
+          @Override
+          public void onCameraMove() {
+            if (mapboxMap.getCameraPosition().zoom > 16) {
               try {
```

Comment:
This turf exception can be removed, it's now runtime

Reply:
removed